### DISCUSSION TITLE
setting up resources during installation

### DISCUSSION THREAD

**Discussion by lexesco99** [splunk/splunk-operator#1145] [Q&A]
When installing Splunk using Splunk operator I see there's an option set the amount of search heads, indexers, and etc..... Is there an option to set the amount of cores and data partition for the indexers?

**Comment by dev-slatto** [comment]
Hi @lexesco99 !
You are able to implement taints on what nodes you should use under the hood for the operator/indexer. You can also specify your overall resource requests and limits within the manifest, but these goes on each pod that's deployed and not your overall indexer. In that case you would have to use the taints. The way you would implement this is to have a node group with your desired nodes that fulfill your requirements, and then configure this node group with a taint label that you can then reference again on the indexer manifest. Please let me know if anything was unclear, and if not, feel free to mark this as a resolution for others that might have the same question 😊

**Comment by wzhiana** [comment]
yep, `limits` works fine for memory & cpu

for disks not so good, but you can do tricks. 
for example we are using ephemeral storage for indexers and host-path mounted dirs + SPLUNK_DB variable to point to it.

**Comment by akondur** [comment]
Hey @lexesco99 , did the answers above resolve the question?
